apps/microtvm/zephyr/demo_runtime/src/main.c

```diff
@@ -214,7 +214,8 @@ tvm_crt_error_t TVMPlatformTimerStop(double* elapsed_time_seconds) {
 }
 
 // Ring buffer used to store data read from the UART on rx interrupt.
-#define RING_BUF_SIZE_BYTES 4 * 1024
+// Should be larger than TVM_CRT_MAX_PACKET_SIZE_BYTES
```

Comment:
can you comment how much larger, and why?

Reply:
the minimum value is 25 bytes and I don't know why.
Also I tested with different baudrates (default is 115200) to see if that solve the issue, but it didn't help.